src/main/kotlin/app/revanced/patches/duolingo/unlocksuper/fingerprints/UserSerializationMethodFingerprint.kt

```diff
@@ -0,0 +1,14 @@
+package app.revanced.patches.duolingo.unlocksuper.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+
+object UserSerializationMethodFingerprint : MethodFingerprint(
+    returnType = "V",
+    accessFlags = AccessFlags.PUBLIC or AccessFlags.CONSTRUCTOR,
+    strings = listOf("betaStatus", "coachOutfit", "globalAmbassadorStatus"),
+    customFingerprint = { m, _ ->
+        m.annotations.isNotEmpty()
+    }
```

Comment:
Is this necessary?

Reply:
Yes I think so, there are 3 files with the same return type, access flags and also strings

only one of them has annotations - I know this is not ideal but it's the best way I've found so far (it also has a different set of parameters, but that's even more likely to change)